oauth2/src/main/java/com/okta/spring/oauth/ConfigurableAccessTokenConverter.java

```diff
@@ -74,7 +77,16 @@ public OAuth2AccessToken extractAccessToken(String value, Map<String, ?> map) {
 
     @Override
     public OAuth2Authentication extractAuthentication(Map<String, ?> map) {
-        return super.extractAuthentication(tweakScopeMap(map));
-    }
+        OAuth2Authentication originalOAuth2Authentication = super.extractAuthentication(tweakScopeMap(map));
 
```

Comment:
Do you think we need some comments/javadoc for this?

Reply:
👍